src/core/core.scale.js

```diff
@@ -1272,6 +1276,7 @@ class Scale extends Element {
 		for (i = 0, ilen = items.length; i < ilen; ++i) {
 			const item = items[i];
 			const tickFont = item.font;
+			const useTextWithStroke = tickFont.lineWidth > 0 && tickFont.strokeStyle !== '';
```

Comment:
The variable name sounds like it's referring to the text instead of the stroke. How about just calling it `useStroke`?

Reply:
changed